### PR TITLE
feat: Disable messages with API_MESSAGES

### LINKED ISSUE #80: Disabling GET/POST on absent /__messages__ endpoint
Good day,

I have commented out the following bunch of lines from support.js in order to avoid 401 errors on the server, where the person responsible for it is not willing to implement  _ **_yet_**_ the [@bahmutov/all-logs](../../all-logs) tool at this stage.

> Lines [35](../blob/master/support.js#L35)-[40](../blob/master/support.js#L40), lines [100](../blob/master/support.js#L100)-[163](../blob/master/support.js#L163) and line [189](../blob/master/support.js#L189)

Result before commenting:
401__messages__

Could there be another way to make it an option, available before loading the cypress script?

I would rather leave cy-api's files untouched 😄

Thanks for any piece of advice

Respectfully

### PR DESCRIPTION
**Pull request by LeJeanbono**
Fix #80

### CODE REVIEW

**Comment by LeJeanbono** [comment]
@bahmutov Can you take a look please :)

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ import '@bahmutov/cy-api/support'
 
 This will add a new command `cy.api` for making API requests.
 
+## Configuration
+
+If you want to disable messages calls : `export CYPRESS_API_MESSAGES=false`
```
Comment (bahmutov) [inline README.md:31]:
```suggestion
If you want to disable messages calls use an environment variable `CYPRESS_API_MESSAGE=false`.
```

**Review by bahmutov — APPROVED** [review]
nice, merging it

**Pull request merged**